abitrolly/sourcegraph
# Disable scheduled workflows in forks

## Test plan

For repo, wait a day and notice there are no these headers in your mailbox.

* Run failed: PR Preview - main
* Run failed: Progress bot - main
* Run failed: Tracking Issue Syncer - main
* Run failed: buildchecker - main
